app/assets/javascripts/discourse/app/controllers/user-activity.js

```diff
@@ -28,6 +28,13 @@ export default Controller.extend({
     this.set("application.showFooter", showFooter);
   },
 
+  @discourseComputed("currentUser.draft_count")
+  draftLabel(count) {
+    return count > 0
+      ? I18n.t("drafts.label_with_count", { count })
+      : I18n.t("drafts.label");
```

Comment:
Why don't you use a single string to handle all the counts?

Reply:
I noticed that we do not show the count when it is 0. We prefer to show "Drafts" over "Drafts (0)".

Reply:
We can just have the translation do that.

Example

```yaml
drafts:
  label:
    zero: "Drafts"
    other: "Drafts (%{count})"
```

Reply:
Ooooh, that's what you meant. I forgot we can do that.

Reply:
Unfortunately, this is not possible because of the pluralization rules for English.